src/main/java/MassimoBoi/MassimoBoi.java

```
@@ -0,0 +1,205 @@
+package MassimoBoi;
+
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import MassimoBoiException.*;
+
+public class MassimoBoi {
+    public static void main(String[] args) throws IOException {
+
+        String logo = " __  __               _                   _           _ \n" +
+                "|  \\/  | __ _ ___ ___(_)_ __ ___   ___   | |__   ___ (_)\n" +
+                "| |\\/| |/ _` / __/ __| | '_ ` _ \\ / _ \\  | '_ \\ / _ \\| |\n" +
+                "| |  | | (_| \\__ \\__ \\ | | | | | | (_) | | |_) | (_) | |\n" +
+                "|_|  |_|\\__,_|___/___/_|_| |_| |_|\\___/  |_.__/ \\___/|_|";
+
+        System.out.println("Hello from\n" + logo);
+        Scanner myObj = new Scanner(System.in);
+        String userInput = "";
+        List<Task> taskList = new ArrayList<>();
+        try{
+            FileClass.loadList(taskList);
+        } catch(FileNotFoundException e){
+
```

try not to leave empty catch blocks